views/calculator.erb

```diff
@@ -2,23 +2,23 @@
 <html>
 <head>
     <title>Calc-U-Later</title>
-    <!--  make stylesheet -->
+    <link rel="stylesheet" type="text/css" href="style.css">
 </head>
-<body style="font-family: sans-serif; text-align: center;">
+<body>
     <h1>Calc U Later!</h1>
-    <div style="margin: 0 auto; background-color: gold; padding: 50px; border-radius: 10px; max-width: 600px;">
+    <div class="container">
         <form action="/" method = "post">
-            <input type="text" name="first_value" value=<%= @first_number %> style="width: 100px; height: 70px; font-size: 34px; text-align: center; border-radius: 5px; border: 1px solid gray;" autofocus></input>
-            <select name="operator" style="width: 85px; height: 75px; background: white; line-height: 75px; font-size: 34px; text-align: center; border: 1px solid gray;" >
+            <input type="text" name="first_value" class="input-box" value="<%= @first_value %>" autofocus></input>
```

Comment:
@pynkvalerian @hewrin FYI I added the quote marks for `@first_value` and `@second_value` below